package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "evm-lite-env",
+  "version": "0.1.0",
+  "description": "Deploy environment for running Lightweight Ethereum node with interchangeable consensus.",
```

Comment:
It is not Ethereum node anymore. Instead, we would call it EVM node since it is now decoupled from Ethereum ecosystem.

Reply:
Okay, I have made the word more precise in a new commit.